charts/prefect-server/values.yaml

```diff
@@ -65,14 +59,11 @@ server:
   # sets PREFECT_LOGGING_SERVER_LEVEL
   loggingLevel: WARNING
 
-  ## If you want to run the UI seperately from the server, you will need to set the UI-specific URLs below
   uiConfig:
     # -- set PREFECT_UI_ENABLED; enable the UI on the server
     enabled: true
-    # -- sets PREFECT_UI_API_URL
-    prefectUiApiUrl: ""
-    # -- sets PREFECT_UI_URL
-    prefectUiUrl: ""
```

Comment:
this was used solely to display the connection url in the `notes.txt`. I've updated this to infer the UI url from the `prefectUiApiUrl`